sysutils/pfSense-pkg-LCDproc/files/usr/local/www/packages/lcdproc/lcdproc.php

```diff
@@ -334,6 +336,17 @@ function updateInputVisibility() {
 	'LED4: Gateway status (green: ok, red: at least one gateway not responding, off: no gateway configured).'
 );
 
+$section->addInput(
+	new Form_Checkbox(
+		'controlmenu', // checkbox name (id)
+		'pfSense control menu', // checkbox label
+		'Enable the pfSense control menu next to LCDproc\'s Options menu.', // checkbox text
+		$pconfig['controlmenu'] // checkbox initial value
+	)
+)->setHelp(
+	'This will only be usefull on display with buttons.<br />' .
+	'Currently you can REBOOT and HALT the system from there.'
+);
```

Comment:
Please remove `<br />` from the Help string and use a parameter like used in `printf()`